recon-enumeration/activities/not-so-random/README.md

```diff
@@ -1,14 +1,8 @@
 # Name: Not So Random
```

Comment:
Please use the same naming convention for the challenges as mentioned here. [1]
From what I saw for the 07 session you did in this way. (<Web/Binary>: <session_name>: <activity_literal_name> )


[1]: https://docs.google.com/document/d/1qejQpcb4K5WoJaga0SJa78AxNh5F_Pjea2rX6DeZmIc/edit#